test/compound-rate-keeper/CompoundRateKeeperV2.test.js

```diff
@@ -0,0 +1,234 @@
+const { assert } = require("chai");
+const { toBN, decimal, fromDecimal } = require("../../scripts/helpers/utils");
+const { setNextBlockTime } = require("../helpers/hardhatTimeTraveller");
+
+const truffleAssert = require("truffle-assertions");
+const Reverter = require("../helpers/reverter");
```

Comment:
We are not using reverter as of now

Reply:
Umm, not quite clear, are we using something else for snapshots and time-travels?

Reply:
Take a look at the structure of the other tests

Reply:
It's still not clear. There are no snapshots in the other tests.

Reply:
Exactly, that is why we don't need it here either

Reply:
So what to use for time-based testing, given that there are no other similar tests in the current repository?